docs/admin/kubeadm-alpha.md

```diff
@@ -0,0 +1,266 @@
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase mark-master {#cmd-phase-mark-master}
+
+Use the following command to label the node with the `node-role.kubernetes.io/master` taint.
+
+{% capture mark-master %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_mark-master.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-mark-master" %}
+{% assign tab_names = "mark-master" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: mark-master %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase bootstrap-token {#cmd-phase-bootstrap-token}
+
+Use the following actions to fully configure bootstrap tokens.
+
+**Note:** You have to run all commands sequentially to replicate the `kubeadm init` behaviour. 
+{: .note}
+
+{% capture bootstrap-token_cluster-info %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_bootstrap-token_cluster-info.md %}
+{% endcapture %}
+
+{% capture bootstrap-token_node_allow-auto-approve %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_bootstrap-token_node_allow-auto-approve.md %}
+{% endcapture %}
+
+{% capture bootstrap-token_node_allow-post-csrs %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_bootstrap-token_node_allow-post-csrs.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-bootstrap-token" %}
+{% assign tab_names = "cluster-info,node allow-auto-approve,node allow-post-csrs" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: bootstrap-token_cluster-info | push: bootstrap-token_node_allow-auto-approve | push: bootstrap-token_node_allow-post-csrs %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase upload-config {#cmd-phase-upload-config}
+
+You can use this command to upload the configuration of your cluster. Alternatively, you 
+can use [kubeadm config](kubeadm-config.md) as well.
+
+{% capture upload-config %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_upload-config.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-upload-config" %}
+{% assign tab_names = "upload-config" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: upload-config %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase addon {#cmd-phase-addon}
+
+You can install all the available abbons with the `all` sub command, or 
+install them selectively.
+
+{% capture addon-all %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_addon_all.md %}
+{% endcapture %}
+
+{% capture addon-kube-proxy %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_addon_kube-proxy.md %}
+{% endcapture %}
+
+{% capture addon-kube-dns %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_addon_kube-dns.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-addon" %}
+{% assign tab_names = "all,kube-proxy,kube-dns" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: addon-all | push: addon-kube-proxy | push: addon-kube-dns %}
+
+{% include tabs.md %}
+
+## Kubeadm alpha phase selfhosting {#cmd-phase-self-hosting}
```

Comment:
If I look around the web, I see self hosting and self-hosting, but not the single word selfhosting. I think we should use this convention:
Noun: self hosting
Adjective: self-hosting
Adjective: self-hosted

So for this heading, I would recommend "kubeadm alpha phase self hosting".